### ISSUE TITLE
vl is undefined (under ES6)

### ISSUE THREAD

**Issue by jazzido** [vega/vega-tooltip#110]
Hi,

I'm trying to use `vega-tooltip` [on an ES6 project](https://github.com/jazzido/mondrian-rest-ui/blob/vega-lite-2.0/src/components/ChartContainer.js#L215-L227), but one of the functions can't find the global reference to `vl` (vega lite) object.

```
TypeError: Cannot read property 'spec' of undefined
    at Object.supplementOptions (vega-tooltip.js:2831)
    at vegaLite (vega-tooltip.js:2432)
    at ChartContainer.js:225
```

I'm guessing this has to do with how `vega-tooltip` is compiled down to javascript. 

Any clues?

Thanks!

**Comment by domoritz** [comment]
Can you provide some code for us to reproduce this issue?

**Comment by sirahd** [comment]
@jazzido did you install all the dependencies, especially to the newest version of `vega-lite`?

**Comment by jazzido** [comment]
Thenks @domoritz and @sirahd 

> Can you provide some code for us to reproduce this issue?

I just created a minimal repro case here: https://github.com/jazzido/vega-tooltip-repro-110

> did you update all the dependencies, especially to the newest version of vega-lite? The cause might also be that we compile the target to ES5.

The chart is displayed correctly, both in the repro code and in my project (see versions in [`package.json`](https://github.com/jazzido/vega-tooltip-repro-110/blob/master/package.json))

Thanks!

**Comment by domoritz** [comment]
Again, thanks for such an easy way to reproduce the issue. I believe https://github.com/vega/vega-tooltip/pull/112 fixes the problem.

**Comment by jazzido** [comment]
Thanks :) Any plans to cut a new release soon?

**Comment by domoritz** [comment]
Just released v0.4.0
